filetests/regalloc/constraints.clif

```diff
@@ -61,12 +60,10 @@ ebb0:
 ; Tied use of a diverted register.
 function %fixed_op_twice() -> i32 {
 ebb0:
-    ; check: ,%rax]
```

Comment:
I'm pretty sure this change violates the intent of this test: that `v0` is in some register and must be diverted for multiple uses. Passing along the `FixedReg(rcx)` affinity ends up placing `v0` directly in rcx, so this no longer tests diversions as it originally did.

I think it would preserve the spirit of this test to have some value, `vN` used as a shift parameter, so it's placed in `rcx`, and subsequently used somewhere as _not_ the shift parameter twice. If that all sounds right, I'll adjust the test.

Reply:
Realized while cleaning up the branch today it's much easier to preserve the intent of this test by forcing `v0` into `rax` by having it be the return value of a function. So it's that now, instead of expecting `v0` to be in the first available register.